__tests__/app.test.js

```diff
@@ -116,6 +132,41 @@ describe("GET /api/articles/:article_id", () => {
   });
 });
 
+describe("GET /api/articles/:article_id/comments", () => {
+  test("200: Returns all comments based on article_id, descending order", () => {
+    return request(app)
+      .get("/api/articles/1/comments")
+      .expect(200)
+      .then(({ body: { comments } }) => {
+          expect(comments.length).toBeGreaterThan(0);
```

Comment:
well odne for asserting against the length here to make sure we dont get a false pass!